bokehjs/src/lib/core/layout/side_panel.ts

```diff
@@ -233,10 +233,7 @@ export class SidePanel extends ContentLayoutable {
       baseline = _baseline_lookup[side][orient]
       align = _align_lookup[side][orient]
     } else {
-      if (orient === 0) {
-        baseline = "whatever" as any // XXX: _baseline_lookup[side][orient]
-        align = "whatever" as any // XXX: _align_lookup[side][orient]
-      } else if (orient < 0) {
```

Comment:
@bryevdv, I don't recall why this special case is here, but this has been broken since I can remember. Maybe there are better defaults for this case?

Reply:
I'd love to know what the rationale was but it was the better part of a decade ago and I really have no idea. The original CS (at least as far back as I can find) does this:
```
     else if orient == 0
        baseline = _baseline_lookup[side][orient]
        align = _align_lookup[side][orient]
```
which is clearly just broken unless CS does something magical with `obj[0]` when there is no `0` key in the object. Seems safe to remove with some minimal smoke testing.